### ISSUE TITLE
NatNet Parser is outdated

### ISSUE THREAD

**Issue by LucieDance** [hku-ect/NatNet2OSCbridge#115]
Hello, I have tried to run your application on Mac OS as well as on Windows and when I get it run it does says it is connected but this error is coming up: "The Implemented NatNet parser is outdated". 

I have tried the Python version too and it does not re-connect. 

I am working with Motive 2.0. version. Could this be an issuew that I am not using lates Motive? or issue with Nat SDK? 

Any suggestions to fix it would be greatful.

**Comment by sphaero** [comment]
Sorry to respond so late. Our versions of Motiv are 1.9 based. We don't have any newer so we can't test these. I know there are newer versions with a slightly different NatNet protocol. Have had any success?

**Comment by LucieDance** [comment]
Hello, thank you. Yes, success with the Motive 1.9. Just wanted to ask the new release from Jun 2020 does this work with the Motive 2.1 and possibly 3.0? Thank you.

**Comment by sphaero** [comment]
No, we are still using the same Motive version (1.9). We've tried newer but this requires a lot of tweaks of ofxNatNet. I haven't searched yet if somebody has an updated ofxNatNet. Eventually we will have to go through this.
